modules/bbp-core/src/main/resources/contexts/bbp/core/entity/v0.1.0.json

```diff
@@ -4,6 +4,9 @@
     "{{base}}/contexts/nexus/core/resource/v0.1.0",
     "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
     {
+      "property": {
```

Comment:
should this be added to the nsg context? I.e. to {{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0